### ISSUE TITLE
I look forward to comparing data   https://reopentexas.info/

### ISSUE THREAD

**Issue by ReopenTexas** [ryankemper/writings#14]
I've only gotten to skim your work, but I am very excited looking forward to reading it in detail tomorrow....that life stuff gets in the way.
I put together an analysis to try to get Texas ASAP.  Mine is posted at   reopentexas.info

If you can use anything from it, please feel free.  All sources are linked.

**Comment by ryankemper** [comment]
Thanks. I’ll be doing the next iteration of integrating research findings
tomorrow (Saturday), so I’ll take a look at your doc and let you know my
thoughts around the same time. I skimmed it and it looks like a very
sensible policy proposal.

Glad to see I’m not the only citizen trying to outline a decision framework
in the absence of (I feel) proper leadership from our elected officials and
policy setters :)

Ryan

**Comment by ryankemper** [comment]
@ReopenTexas Not sure if you've got email notifications enabled, but overall I think the proposal is very good. I haven't given it a thorough read, but this caught my eye:

> but the author has personal knowledge of a Houstonian who became seriously ill with classic COVID-19 symptoms after returning home from Thanksgiving, 2019 in Farmington, NM.There have been other reports of higher than normal, serious flu-like illnesses on the West Coast in late 2019.)

We should be careful with this, because there are a *lot* of diseases that cause Influenza-like symptoms. Not just Influenza itself, but adenoviruses, rhinoviruses, and a bunch more that I don't know a lot about.

We obviously cannot say for sure, but it seems likely that there were no cases of COVID-19 in the US until 2020. I would bet genomic analysis can give us a pretty good idea of the timeline, but I haven't touched much of that research literature yet.

[This is the classic study that describes the first known COVID-19 case in the US](https://www.nejm.org/doi/full/10.1056/NEJMoa2001191) around mid january 2020. Here's the timeline:

```
    dec 31, china reports cluster of pneumonia cases
    jan 7,  china alerts health authorities that cluster is associated with SARS-CoV-2 (at the time it was called 2019-nCoV)
    jan 20, first reported us case (they had active symptoms for a few days prior)
```

BTW, this shows how many other things can cause similar symptoms:

> A nasopharyngeal swab specimen was obtained and sent for detection of viral respiratory pathogens by NAAT; this was reported back within 48 hours as negative for all pathogens tested, including influenza A and B, parainfluenza, respiratory syncytial virus, rhinovirus, adenovirus, and four common coronavirus strains known to cause illness in humans (HKU1, NL63, 229E, and OC43).

And that's only what we can feasibly test for!

So in conclusion, I think it's best to not include that speculation.

I'm totally behind the overall proposal. We share the same central theme:

> There is a large percentage of the NYC populationthat have been only slightly affected by COVID-19 butare almost all of them are affected by the government shutdown of the city.

On a different note, I'm interested in the idea you mentioned that age is only predictive of mortality insofar as it's correlated with comorbidities. I haven't investigated whether that's true myself, but that's an interesting thought.

**Comment by ReopenTexas** [comment]
Thanks for the feedback Ryan!I hope you get this.  I've never used GitHub before, so not exactly sure how it works.  :)


-------- Original Message --------
Subject: Re: [ryankemper/writings] I look forward to comparing data  
https://reopentexas.info/ (#14)

**Comment by ryankemper** [comment]
Yup, I've got e-mail notifications enabled so I'll see any messages coming through. BTW if it's easier my e-mail is listed on https://www.ryankemper.io/ (see very bottom left).

Also, since I probably wasn't super clear, note that it is almost certain that we had COVID-19 infections earlier than the earliest documented infection.

At a minimum it was probably spreading at least a month before China notified about the cluster of pneumonia cases on Dec 31.

And we're gradually hearing reports like https://thehill.com/policy/international/europe/496075-french-hospital-discovers-it-treated-coronavirus-patient-in. So just to be clear - while I still find it hard to believe that it was truly in the US in November, I'm not saying that it's not possible, but more-so that it's a very easy target for those who might seek to discredit you to latch onto.

Hope things are going well in your neck of the woods

**Comment by ReopenTexas** [comment]
Awesome.  Thanks for the clarification!


-------- Original Message --------
Subject: Re: [ryankemper/writings] I look forward to comparing data  
https://reopentexas.info/ (#14)
